### DISCUSSION TITLE
Does this have a docker container for easier depoloyment?

### DISCUSSION THREAD

**Discussion by tristone13th** [levito/tt-rss-feedly-theme#112] [Q&A] [answered]
If the ttrss is installed using docker, then it is impossible to copy the css into the directory, so does this plugin have a docker container?

**Comment by levito** [comment, marked as answer]
No, sorry, I didn't care about docker so far. I'm using tt-rss straight from git. ~If anybody can help making this theme docker friendly, please don't hesitate to create a PR!~

Edit:
Here are the docs on how to add themes to the dockerized tt-rss:
https://git.tt-rss.org/fox/ttrss-docker-compose.wiki.git/tree/Home.md#how-do-i-add-plugins-and-themes
